manifests/o/Opera/OperaCrypto/94.0.4606.43/Opera.OperaCrypto.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Opera.OperaCrypto
+PackageVersion: 94.0.4606.43
+PackageLocale: en-US
+Publisher: Opera Software
+PackageName: Opera Crypto Stable
+License: Proprietary software

```

Comment:
```suggestion
PublisherUrl: https://www.opera.com/
PublisherSupportUrl: https://www.opera.com/help
PrivacyUrl: https://legal.opera.com/privacy/
PackageName: Opera Crypto Stable
PackageUrl: https://www.opera.com/crypto/next
License: Proprietary software
LicenseUrl: https://legal.opera.com/eula/computers/
```

Reply:
I have made these changes and added some more